src/components/patients/searchTool/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Icon, H3, Button } from "@blueprintjs/core";
+import "./style.css";
+import SearchItem from "components/patients/searchTool/searchItem";
+
+interface searchForm {
+  attribute: string;
+  symbol: string;
+  text: string;
+}
+
+const SearchTool = () => {
+  const [searchItems, setSearchItems] = React.useState([0]);
+  const [searchForms, setSearchForms] = React.useState([] as searchForm[]);
```

Comment:
I think it would make things easier to use an object instead of an array to avoid dealing with indexes.